### PR TITLE
Move progressions strings/translations to yml

### PR DESCRIPTION
**Pull request by bethanyaconnor**
It turns out that we rely on Crowdin to modify the language key in our yml files. Because the progressions translations do not need to be modified before distribution, let's continue to rely on Crowdin doing this work for us.

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: bin/i18n/sync-out.rb [hunk 1]
```diff
@@ -260,17 +260,13 @@ def distribute_translations(upload_manifests)
     next unless File.directory?(locale_dir)
 
     ### Dashboard
-    Dir.glob("i18n/locales/#{locale}/dashboard/*.{json,yml}") do |loc_file|
```
Comment (Hamms) [inline bin/i18n/sync-out.rb]: I'd suggest we keep these changes in place, to smooth over any possible future efforts to switch back to json

**Review by Hamms — APPROVED** [review]
I think this decision makes sense. I do still want to switch us over to JSON as a long-term goal, but I think it makes more sense to do that first for an existing feature, rather than trying to navigate all the potential issues with that change within the context of a new feature.

**Comment by bethanyaconnor** [comment]
Yeah, that was my thought too. I'd rather any potential issues with this feature be for the feature itself instead of the translation format

File: bin/i18n/sync-out.rb (hunk 1 above)
Comment (bethanyaconnor) [reply bin/i18n/sync-out.rb]: Sounds good. I'll pull in the fix from #35929 while I'm at it as well then.

**Pull request merged**